mmv1/products/cloudrun/terraform.yaml

```diff
@@ -73,8 +73,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         name: "cloud_run_service_tasks"
         primary_resource_id: "default"
         vars:
-          project_id: "project-id"
-          region: "us-central1"
+          skip_docs: true
```

Comment:
i think we want `skip_docs` at the same level as `vars`, not underneath it. also, i'm a little confused why we're wanting to skip the docs if the intention was to create samples? did you mean `skip_test`? thanks!

Reply:
Thank you for catching this! Apologies for all the back and forth. Reason for adding is, within our team, since we are only designing this for the quickstarts - we didn't feel comfortable airing this on the registry.hashicorp site just yet. If that's alright, happy to discuss more tho!